pyproject.toml

```
@@ -9,7 +9,7 @@ readme = "README.rst"
 
 [tool.poetry.dependencies]
 python = "^3.9"
-httpx = "^0.23.3"
+httpx = ">=0.23.3 <4"
```

```suggestion
httpx = ">=0.23.3"
```

Just wondering why not this?

---

That was a typo on my part, I wanted to say `<1`, not `<4`.

I updated the proposed change so that it doesn't include an upper limit.